File: json/R/create/json_create.R
# ---------------------------------------------------------
#	json_create.R
#
#				Feb/21/2014
#
# ---------------------------------------------------------
record_create_proc<-function (id,name,population,date_mod,tx){
#
	write (paste ("\"",id,"\": {",sep = ""),file=file_out, append=T)
	write (paste ("\"name\":\"",name,"\"," ,sep = ""),file=file_out, append=T)
	write (paste ("\"population\":",population,",",sep = ""),file=file_out, append=T)
	write (paste ("\"date_mod\":\"",date_mod,"\"}",tx,sep = ""),file=file_out, append=T)

}

# ---------------------------------------------------------
print ("*** 開始 ***")
file_out <- commandArgs(trailingOnly=TRUE)[2]
print (file_out)

write ("{",file=file_out)

record_create_proc ("t0921","宇都宮","49817","1946-8-14",",")
record_create_proc ("t0922","小山","75124","1946-9-27",",")
record_create_proc ("t0923","佐野","35891","1946-5-21",",")
record_create_proc ("t0924","足利","87365","1946-7-15",",")
record_create_proc ("t0925","日光","28564","1946-10-5",",")
record_create_proc ("t0926","下野","51297","1946-1-17",",")
record_create_proc ("t0927","さくら","36718","1946-3-18",",")
record_create_proc ("t0928","矢板","47386","1946-8-9",",")
record_create_proc ("t0929","真岡","39845","1946-2-22",",")
record_create_proc ("t0930","栃木","51827","1946-6-21",",")
record_create_proc ("t0931","大田原","45763","1946-4-8",",")
record_create_proc ("t0932","鹿沼","81462","1946-8-7",",")
record_create_proc ("t0933","那須塩原","63829","1946-7-12",",")
record_create_proc ("t0934","那須烏山","27651","1946-3-21","}")


print ("*** 終了 ***")
# ---------------------------------------------------------

File: json/clojure/create/json_create.clj
; ------------------------------------------------------------------
;
;	json_create.clj
;
;					Jul/03/2011
;
; ------------------------------------------------------------------
(use '[clojure.contrib.duck-streams])
;(use '[clojure.data.json.read])
;(use '[clojure.data.json.write])
;(use 'clojure.data.json)
(use 'clojure.data.json.read)
; ------------------------------------------------------------------
(defn data_prepare_proc []
{"t0921" {:name "宇都宮" :population 34657 :date_mod "1970-1-5"},
"t0922" {:name "小山" :population 65718 :date_mod "1970-4-15"},
"t0923" {:name "佐野" :population 87594 :date_mod "1970-3-8"},
"t0924" {:name "足利" :population 79182 :date_mod "1970-6-12"},
"t0925" {:name "日光" :population 68263 :date_mod "1970-11-29"},
"t0926" {:name "下野" :population 51475 :date_mod "1970-9-25"},
"t0927" {:name "さくら" :population 34528 :date_mod  "1970-8-7"},
"t0928" {:name "矢板" :population 42179 :date_mod  "1970-10-25"},
"t0929" {:name "真岡" :population 24937 :date_mod  "1970-6-15"},
"t0930" {:name "栃木" :population 73891 :date_mod  "1970-9-22"},
"t0931" {:name "大田原" :population 27157 :date_mod  "1970-3-14"},
"t0932" {:name "鹿沼" :population 47251 :date_mod  "1970-7-29"},
"t0933" {:name "那須塩原" :population 43827 :date_mod  "1970-5-4"},
"t0934" {:name "那須烏山" :population 98538 :date_mod  "1970-2-18"}}
)
; ------------------------------------------------------------------
(println "*** 開始 ***")
(def dict_aa (data_prepare_proc))
;
(spit (first *command-line-args*) (json-str dict_aa))
(println "*** 終了 ***")
; ------------------------------------------------------------------
